app/src/main/java/ch/epfl/sdp/appart/map/ApartmentInfoWindow.java

```diff
@@ -60,7 +60,7 @@ public View getInfoContents(Marker marker) {
         priceTextView.setText(card.getPrice() + " CHF");
         if (card.getImageUrl() != null) {
             databaseService.accept(new GlideImageViewLoaderListener(activity,
-                    photo, "Cards/" + card.getImageUrl(),
+                    photo, "ads/" + card.getAdId() + "/" + card.getImageUrl(),
```

Comment:
I suggest to use the constant values defined in `FirebaseLayout`

Reply:
Yes, I forgot about that. The issue with this, is that it leaks the underlying database. It is not really important in this particular class but maybe we should give a more neutral name to the file. But for now, I don't think it is that important and it is still better to use the constants in FirebaseLayout than to hardcode strings.